packages/devextreme-vue-generator/src/generator.ts

```diff
@@ -67,13 +67,16 @@ function generate(
   writeFile(out.indexFileName, generateIndex(modulePaths), { encoding: "utf8" });
 
   if (generateReexports && rawData.commonReexports) {
-    const commonPath = joinPaths(out.componentsDir, "common");
+    const commonTargetFolderName = "common";
+    const commonPath = joinPaths(out.componentsDir, commonTargetFolderName);
     if (!existsSync(commonPath)) {
       mkdirSync(commonPath);
     }
     Object.keys(rawData.commonReexports).forEach((key) => {
+      const targetKey = key.replace(`${commonTargetFolderName}/`, "");
+      const targetFileName = `${targetKey === commonTargetFolderName ? "index" : targetKey}.ts`;
```

Comment:
**Minor**
To avoid using ternary within string:
```suggestion
      const targetFileName =  targetKey === commonTargetFolderName
        ? 'index.ts'
        : `${targetKey}.ts`;
```

Reply:
Maybe, the condition can be changes in this way:

```suggestion
      const targetFileName =  key === commonTargetFolderName
        ? "index.ts"
        : `${key.replace(`${commonTargetFolderName}/`, "")}.ts`;
```

Reply:
fixed